site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -157,27 +211,33 @@ all mouse events.
 3.  Check one of these categories to pause whenever any event from that category is fired, or expand
     the category and check a specific event.
 
-{% Img src="image/admin/c40PakNjXuOPIn21VySn.png", alt="Creating an event listener breakpoint.", width="800", height="543" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/aFFgOPi3outuMb32WO3k.png", alt="Creating an event listener breakpoint.", width="800", height="726" %}
```

Comment:
Somewhere in this section mention the event listener breakpoint can be found in Elements panel > Event Listeners too

Reply:
Ummm... Settings such a breakpoint in Sources doesn't populate the Elements > Event Listener list. Maybe because it's listeners, not breakpoints?

Reply:
Oh yes, it Elements > Event Listener list only list the listeners, not breakpoints.